packages/react-core/src/components/SimpleList/examples/SimpleListUncontrolled.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { SimpleList, SimpleListItem } from '@patternfly/react-core';
+
+export const SimpleListUncontrolledDemo: React.FunctionComponent = () => {
+  const [activeItem, setActiveItem] = React.useState(0);
+
+  const onSelect = (
+    selectedItem: React.RefObject<HTMLButtonElement> | React.RefObject<HTMLAnchorElement>,
+    _selectedItemProps
```

Comment:
I think the type of the selectedItemProps is `SimpleListItemProps`. 

We usually use the underscore at the beginning of parameters as a cue that the parameter is not going to be used in the function body. In this case, the selectedItem is actually not used in the body, but the selectedItemProps param is, so i'd remove the underscore from the param name.

Reply:
got it - thanks